src/rules/effects/prefer-concat-latest-from.ts

```diff
@@ -33,6 +36,10 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
   },
   defaultOptions: [],
   create: (context) => {
+    if (!ruleEnabled) {
```

Comment:
1. Would it be possible to us to test this behavior? Maybe mocking the `require('@ngrx/effects/schematics-core')`?
2. What about `console.warn` (or something like that) informing the user that the rule isn't supported given XX version of `@ngrx`?

Reply:
1. Would it be possible to us to test this behavior? Maybe mocking the `require('@ngrx/effects/schematics-core')`?
2. What about `console.warn` (or something like that) informing the user that the rule isn't supported given XX version of `@ngrx`?